pkgs/top-level/python-packages.nix

```diff
@@ -8710,6 +8722,16 @@ in {
 
   tensorflowWithCuda = self.tensorflow.override {
     cudaSupport = true;
+    # See https://github.com/NixOS/patchelf/pull/256#issuecomment-915617721.
+    # This can be removed once that PR is merged and released.
+    patchelf = pkgs.patchelfUnstable.overrideAttrs(old: {
```

Comment:
Please adjust name.

Reply:
How so?

Reply:
You should set name/pname+verison to reflect the unstable version.

Reply:
Done in https://github.com/NixOS/nixpkgs/pull/137069/commits/c29885ef82a607a0009fe502c6a8ac3f0bc18b70.